dot/network/notifications.go

```diff
@@ -226,14 +234,32 @@ func (s *Service) sendData(peer peer.ID, hs Handshake, info *notificationsProtoc
 			return
 		}
 
-		hs, err := s.readHandshake(stream, decodeBlockAnnounceHandshake)
-		if err != nil {
-			logger.Trace("failed to read handshake", "protocol", info.protocolID, "peer", peer, "error", err)
+		hsTicker := time.NewTicker(handshakeTimeout)
```

Comment:
User timer instead of ticker.
```
time.NewTimer(time.Second)
```

Timers are for when you want to do something once in the future - tickers are for when you want to do something repeatedly at regular intervals